tasks/cephfs/test_client_recovery.py

```diff
@@ -307,19 +308,31 @@ def test_network_death(self):
         self.assertLess(recovery_time, self.ms_max_backoff * 2)
         self.assert_session_state(client_id, "open")
 
+
     def test_filelock(self):
         """
         Check that file lock doesn't get lost after an MDS restart
         """
-        lock_holder = self.mount_a.lock_background()
+        a_version = get_package_version(self.mount_a.client_remote, "fuse")
+        b_version = get_package_version(self.mount_b.client_remote, "fuse")
+        flock_version = "2.9"
+        flockable = False
+        if (a_version >= flock_version and b_version >= flock_version):
```

Comment:
String comparison of versions will fail if fuse rev their version to 2.10.  distutils.version.StrictVersion is probably suitable here, although it doesn't like anything other than numbers and dots, so you might also need to truncate the version string to the leading [0-9\.]+